docs/monitors/expvar.md

```diff
@@ -0,0 +1,206 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# expvar
+
+This monitor exports metrics derived from [expvar](https://golang.org/pkg/expvar/)
+variables in JSON objects retrieved from a HTTP endpoint.
+
+The monitor uses configured JSON paths to get metric and dimension values from the fetched JSON objects.
+[Memstats metrics](https://github.com/signalfx/integrations/blob/master/expvar/docs/expvar_metrics.md)
```

Comment:
Do not link to the integrations repo.  This repo and this document are the authoritative docs for this and we will be migrating this to the integrations repo very soon.